src/Fantomas.Tests/RecordTests.fs

```diff
@@ -1894,3 +1894,27 @@ let parse (checker: FSharpChecker) (parsingOptions: FSharpParsingOptions) { File
         })
     |> Async.Parallel
 """
+
+[<Test>]
+let ``anonymous records with comments on members, 2067`` () =
```

Comment:
Maybe change to `with comments on record fields`, as they are not really members.

Reply:
Fixed on b2550b46813c5f7daece9245b1468abbee7f3307